src/crimson/osd/osd.h

```diff
@@ -121,6 +121,7 @@ class OSD final : public crimson::net::Dispatcher,
 
 public:
   OSD(int id, uint32_t nonce,
+      crimson::os::FuturizedStore* store,
```

Comment:
It's initialized in the constructor, let's make it a reference.

Reply:
@athanatos updated and repushed.